client/README.md

```diff
@@ -0,0 +1,45 @@
+# etcd/client
+
+[![GoDoc](https://godoc.org/github.com/coreos/etcd/client?status.png)](https://godoc.org/github.com/coreos/etcd/client)
+
+## Usage
+
+```
+package main
+
+import (
+	"log"
+
+	"github.com/coreos/etcd/client"
+	"golang.org/x/net/context"
+)
+
+func main() {
+	cfg := client.Config{Endpoints: []string{"http://127.0.0.1:4001"}}
+	c, err := client.New(cfg)
+	if err != nil {
+		log.Fatal(err)
+	}
+	kapi := client.NewKeysAPI(c)
+	resp, err := kapi.Set(context.Background(), "test", "bar", nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+```
+
+## Install
+
+```bash
+go get github.com/coreos/etcd/client
+```
+
+## Caveat
```

Comment:
/cc @robszumski Can you please help to read through to make sure they are understandable english...